### PR TITLE
feat(connector): [Volt] Template generation

### PR DESCRIPTION
**Pull request by swangi-kumari**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
Add template code for Volt Connector

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?
Template code testing not required

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: config/config.example.toml [hunk 1]
```diff
@@ -208,6 +208,7 @@ worldline.base_url = "https://eu.sandbox.api-ingenico.com/"
 worldpay.base_url = "https://try.access.worldpay.com/"
 trustpay.base_url = "https://test-tpgw.trustpay.eu/"
 tsys.base_url = "https://stagegw.transnox.com/"
+volt.base_url = "https://api.sandbox.volt.io/"
```
Comment (SamraatBansal) [inline config/config.example.toml:211]: Place it alphabetically

File: config/docker_compose.toml [hunk 2]
```diff
@@ -126,6 +126,7 @@ worldline.base_url = "https://eu.sandbox.api-ingenico.com/"
 worldpay.base_url = "https://try.access.worldpay.com/"
 trustpay.base_url = "https://test-tpgw.trustpay.eu/"
 tsys.base_url = "https://stagegw.transnox.com/"
+volt.base_url = "https://api.sandbox.volt.io/"
```
Comment (SamraatBansal) [inline config/docker_compose.toml:129]: Place it alphabetically

File: crates/router/src/core/payments/flows.rs [hunk 3]
```diff
@@ -523,6 +528,7 @@ default_imp_for_accept_dispute!(
     connector::Stripe,
     connector::Trustpay,
     connector::Tsys,
+    connector::Volt,
     connector::Opennode,
```
Comment (SamraatBansal) [inline crates/router/src/core/payments/flows.rs:532]: Fix the alphabetical order

File: crates/test_utils/src/connector_auth.rs [hunk 4]
```diff
@@ -77,7 +78,7 @@ impl ConnectorAuthentication {
     /// Will panic if `CONNECTOR_AUTH_FILE_PATH` env is not set
     #[allow(clippy::expect_used)]
     pub fn new() -> Self {
-        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_sample_auth.toml"`
+        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_sample_sample_auth.toml"`
```
Comment (SamraatBansal) [inline crates/test_utils/src/connector_auth.rs:81]: Don't include this change

File: crates/test_utils/src/connector_auth.rs [hunk 5]
```diff
@@ -109,7 +110,7 @@ impl ConnectorAuthenticationMap {
     /// Will panic if `CONNECTOR_AUTH_FILE_PATH` env  is not set
     #[allow(clippy::expect_used)]
     pub fn new() -> Self {
-        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_sample_auth.toml"`
+        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_sample_sample_auth.toml"`
```
Comment (SamraatBansal) [inline crates/test_utils/src/connector_auth.rs:113]: Don't include this change

File: loadtest/config/development.toml [hunk 6]
```diff
@@ -111,6 +111,7 @@ worldline.base_url = "https://eu.sandbox.api-ingenico.com/"
 worldpay.base_url = "https://try.access.worldpay.com/"
 trustpay.base_url = "https://test-tpgw.trustpay.eu/"
 tsys.base_url = "https://stagegw.transnox.com/"
+volt.base_url = "https://api.sandbox.volt.io/"
 wise.base_url = "https://api.sandbox.transferwise.tech/"
 trustpay.base_url_bank_redirects = "https://aapi.trustpay.eu/"
```
Comment (SamraatBansal) [inline loadtest/config/development.toml:116]: Place in alphabetical order

**Review by SamraatBansal — CHANGES_REQUESTED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by SamraatBansal — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**